### PR TITLE
chore: update pr and issue templates

### PR DESCRIPTION
**Pull request by luqven**
Update the PR and issue GitHub templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,28 +1,46 @@
...
 A clear and concise description of what you expected to happen.
 
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
 **Information:**
- - imgix-swift version: [e.g. 2.0.0]
+
+- @imgix/vue version: [e.g. v1.0]
+- browser version: [include link from [https://www.whatsmybrowser.org/](https://www.whatsmybrowser.org/) or details about the OS used and browser version]
```
Comment (sherwinski) [inline .github/ISSUE_TEMPLATE/bug_report.md]: It looks like we accidentally replaced swift with vue here

**Review by sherwinski — CHANGES_REQUESTED** [review]
LGTM minus two small nits

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,28 +1,46 @@
...
-labels: ''
-assignees: ''
-
 ---
 
-Please provide the following information and someone from @imgix/imgix-sdk-team will respond to your issue shortly.
+**Before you submit:**
+
+- [ ] Please read the [contributing guidelines](CONTRIBUTING.md)
+- [ ] Please search through the existing issues (both open AND closed) to see if your issue has been discussed before. Github issue search can be used for this: https://github.com/imgix/vue/issues?utf8=%E2%9C%93&q=is%3Aissue
```
Comment (luqven) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
- [ ] Please search through the existing issues (both open AND closed) to see if your issue has been discussed before. GitHub issue search can be used for this: https://github.com/imgix/imgix-swift/issues?utf8=%E2%9C%93&q=is%3Aissue
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -1,28 +1,46 @@
...
 
 **Describe the bug**
-A clear and concise description of what the bug is.
+A clear and concise description of what the bug is. Please strive to reach the **root problem** of your issue to avoid the XY problem. See more: https://meta.stackexchange.com/questions/66377/what-is-the-xy-problem
 
 **To Reproduce**
-Steps to reproduce the behavior:
+A bug is a _demonstrable problem_ that is caused by the code in the repository. Thus, the contributors need a way to reproduce your issue - if we can't reproduce your issue, we can't help you! Also, please be as detailed as possible.
+
+[a link to a codesandox or repl.it; here is a link to a codesandbox with @imgix/vue installed which can be forked: https://codesandbox.io/s/vue-imgix-base-codesandbox-bhz8n]
```
Comment (luqven) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
[add a link to a codesandox or repl.it]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -1,28 +1,46 @@
...
+
+[alternatively, please provide a code example]
 
-**Expected behavior**
+```js
+// A *self-contained* demonstration of the problem follows...
+// This should be able to be dropped into a file with @imgix/vue installed and just work
```
Comment (luqven) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
// This should be able to be dropped into a file with imgix-swift installed and just work
```

**Review by luqven — COMMENTED** [review]
🤦🏼 apols, let's replace all those `vue` with `imgix-swift`.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -1,28 +1,46 @@
...
 **To Reproduce**
-Steps to reproduce the behavior:
+A bug is a _demonstrable problem_ that is caused by the code in the repository. Thus, the contributors need a way to reproduce your issue - if we can't reproduce your issue, we can't help you! Also, please be as detailed as possible.
+
+[add a link to a codesandox or repl.it]
+
+[alternatively, please provide a code example]
 
-**Expected behavior**
+```js
```
Comment (sherwinski) [inline .github/ISSUE_TEMPLATE/bug_report.md]: super nitpick but there's also this
```suggestion
```swift
```

**Comment by luqven** [comment]
Apols, i didn't realize the other pr was still based on this one 😬

**Review by sherwinski — APPROVED** [review]

**Pull request merged**